src/clients/firewall_stage1_proposal.rb

```diff
@@ -76,7 +76,9 @@ def main
         # Don't override users settings
         SuSEFirewall4Network.prepare_proposal unless SuSEFirewallProposal.GetChangedByUser
 
-        adjust_configuration
+        # this method is not easily mockable in rspec and currently is out of scope
+        # for testing in firewall_stage1_proposal_test.rb
+        adjust_configuration if !Mode.test
```

Comment:
Ugh? Mode.test is not set in rspec, so how it can affect it? Also I do not belive that it is hard to mock in rspec. Can you be more specific why you cannot simple mock it out?